src/more-infos/more-info-climate.html

```diff
@@ -208,6 +238,20 @@
     }.bind(this), 500);
   },
 
+  computeTargetTempHidden: function (stateObj) {
+    return stateObj.attributes.operation_mode.includes('Off');
+  },
+
+  computeHideTempRangeSlider: function (stateObj) {
+    return !stateObj.attributes.operation_mode.includes('Auto');
```

Comment:
We hide the normal temp slider when climate device is in auto mode, and replace it with the range slider.